Apps/Src/Current.c

```diff
@@ -184,11 +185,22 @@ void Task_AmperesMonitor(void *p_arg) {
 
 	bool amperesHasBeenChecked = false;
 
+	CANData_t CanData;
+    CanData.b = 0;
+    CanData.h = 0;
+    CanData.w = 0;
+    CANPayload_t CanPayload;
+    CANMSG_t CanMsg;
+
     while(1) {
         // BLOCKING =====================
         // Update Amperes Measurements
 		Current_UpdateMeasurements();
-
+		CanData.f = (float)Amps_GetReading();
```

Comment:
This function returns an `int32_t` that represents a fixed point number, so you can't cast it like this

Reply:
So would I just return the value of Amps_GetReading()? Or would I have to assign that to an integer and then typecast that variable. Do I have to do the same for all of the other CANData's?

Reply:
Maybe cast it to a float, then divide it by 1000 to get Volts? It doesn't make sense to use a float if we are not using the fraction part of it

Reply:
Also, the Amps pull request has been merged into rtos, so you can merge rtos into this and get it to build